java/lang/Double/DoubleTest.cpp

```diff
@@ -268,6 +268,22 @@ TEST (JavaLang, DoubleValueOf) {
     Double::compare(expectedResult.doubleValue(), actualResult.doubleValue());
 }
 
+TEST (JavaLang, DoubleValueOfWithDoubleInput) {
```

Comment:
The ASSERT's accuracy just 3 number of the fraction part. 
We will use Double::compare(double a, double b) to make a comparasion between floating point number.

My method Double::compare() have the accuracy 8 number of the fraction part until now.